cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
+    config = config:info()
+else
+    config = {
+        status = "uninitialized",
+        alerts = {},
+    }
+end
+
+local config_info = {
+    status = config.status,
+    alerts = {}
+}
+for _, alert in pairs(config.alerts) do
```

Comment:
Nit: `alerts` is an array-like table, we can use `ipairs` here.

Reply:
Fixed.